src/Hosting/SystemdServices/src/Microsoft.Extensions.Hosting.SystemdServices.csproj

```diff
@@ -0,0 +1,15 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <Description>.NET hosting infrastructure for Systemd Services.</Description>
+    <TargetFramework>netstandard2.1</TargetFramework>
+    <GenerateDocumentationFile>true</GenerateDocumentationFile>
+    <PackageTags>hosting</PackageTags>
+    <IsShipping>true</IsShipping>
```

Comment:
Should we consider this for the linux shared framework? That would make it possible to enable it by default in the worker template (assuming it does what `.UseWindowsServices` does and only lights up if actually hosted in systemd).

That would also mean it needs to be in source-build.

Of course, I don't know exactly what (if any?) shared framework the worker template references. Does it reference `Microsoft.AspNetCore.App` or `Microsoft.NETCore.App`? If the latter, the point is moot, it's not going in to that shared fx in this release. (cc @glennc)

Reply:
We don't do this for the windows service so we shouldn't do it here either....